app/assets/javascripts/discourse/app/components/future-date-input.js

```diff
@@ -1,25 +1,26 @@
 import Component from "@ember/component";
 import { action } from "@ember/object";
 import { and, empty, equal } from "@ember/object/computed";
-import { CLOSE_STATUS_TYPE } from "discourse/controllers/edit-topic-timer";
 import buildTimeframes from "discourse/lib/timeframes-builder";
 import I18n from "I18n";
 import { FORMAT } from "select-kit/components/future-date-input-selector";
 
 export default Component.extend({
   selection: null,
   includeDateTime: true,
-  isCustom: equal("selection", "pick_date_and_time"),
+  isCustom: equal("selection", "custom"),
   displayDateAndTimePicker: and("includeDateTime", "isCustom"),
   displayLabel: null,
   labelClasses: null,
   timeInputDisabled: empty("_date"),
+  userTimezone: null,
 
   _date: null,
   _time: null,
 
   init() {
     this._super(...arguments);
+    this.userTimezone = this.currentUser.resolvedTimezone(this.currentUser);
```

Comment:
This API looks a bit weird. Why is `resolvedTimezone` on the `currentUser` object? Shouldn't it be a helper method?

Reply:
Yes, it is weird. It's not the first call, see https://github.com/discourse/discourse/pull/15749#discussion_r794970097. I'll fix it in a separate PR in next days.